package.json

```diff
@@ -39,7 +39,7 @@
     "commander": "^4.0.1",
     "fs-extra": "^0.30.0",
     "js-yaml": "^3.13.1",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
```

Comment:
Can you revert this lodash change, and the other changes to `package-lock.json`? We can handle that in a separate PR.

Reply:
didn't saw this files changing

Reply:
np!